envi/archs/arm/__init__.py

```diff
@@ -8,15 +8,13 @@
 from envi.archs.arm.disasm import *
 
 class ArmModule(envi.ArchitectureModule):
-    #Current ARM version working in - Not fully integrated yet
-    archVersion = ('ARMv7A')
 
     def __init__(self, name='ARMv7A'):
-        self.archVersion = name
         import envi.archs.thumb16.disasm as eatd
         envi.ArchitectureModule.__init__(self, name, maxinst=4)
         self._arch_reg = self.archGetRegCtx()
         self._arch_dis = ArmDisasm()
+        self._arch_dis.setArchMask(name)
```

Comment:
At this time It isn't being set in ThumbModule but when I test ran this setup ArmModule was getting called with Thumb and Thumb16 as types which I wasn't expecting and had to account for in the setter in disasm.py. Perhaps it is time for me to look at the Thumb code and understand what it is doing too so I know where to set it there? I did note in the Thumb module where it needs to be set. I will look through the thumb arch stuff briefly today.